docs/index.md

```diff
@@ -44,6 +44,7 @@ an understanding of the system's architecture and each of the system's underlyin
 
 ## System installation basics
 
+- [Getting started guide](basics)
```

Comment:
Should this link to `basics/index.md`? As is it doesn't work from https://github.com/dimagi/commcare-cloud/blob/369706f1596e57800f9a29b3db43acdb6043b48f/docs/index.md#system-installation-basics

Reply:
Oh interesting. I was writing this with the site (http://dimagi.github.io/commcare-cloud/) in mind, and in that context it works fine (i.e. the corresponding page on my local test server, http://127.0.0.1:4000/basics/, does in fact read from the that `index.md` file). Do you think it's important to make it work in the browsing-docs-code-on-github context?

Reply:
Ah, makes sense.

> Do you think it's important to make it work in the browsing-docs-code-on-github context?

I guess not since the link in github code context takes you to a directory listing, and it's fairly obvious you want to start with `index.md`